tests/test_anthropic.py

```diff
@@ -1,7 +1,7 @@
 import unittest
 from anthropic import AI_PROMPT, HUMAN_PROMPT
 
-import pychatml
+from pychatml.anthropic_converter import Anthropic
```

Comment:
I can’t decide if having two words in a module is right I don’t think I have it seen anywhere

Reply:
It was to avoid clashing with the existing folders `anthropic` etc.

Reply:
also i have seen it elsewhere -- for example `from openai import openai.openai_object`

Reply:
@koogle otherwise we have a name collision which because python is un-resolvable essentially